package.json

```diff
@@ -9,9 +9,9 @@
     "url": "https://github.com/quisquous/cactbot"
   },
   "scripts": {
-    "build": "cross-env TS_NODE_PROJECT=\"tsconfig-for-webpack.json\" webpack --config webpack/webpack.prod.ts",
-    "build-gh-pages": "cross-env TS_NODE_PROJECT=\"tsconfig-for-webpack.json\" webpack --config webpack/webpack.ghpages.ts",
-    "start": "cross-env TS_NODE_PROJECT=\"tsconfig-for-webpack.json\" webpack serve --config webpack/webpack.dev.ts",
+    "build": "cross-env NODE_OPTIONS=\"--loader ts-node/esm\" TS_NODE_PROJECT=\"tsconfig-for-webpack.json\" webpack --config webpack/webpack.prod.ts",
+    "build-gh-pages": "cross-env NODE_OPTIONS=\"--loader ts-node/esm\" TS_NODE_PROJECT=\"tsconfig-for-webpack.json\" webpack --config webpack/webpack.ghpages.ts",
+    "start": "cross-env NODE_OPTIONS=\"--loader ts-node/esm\" TS_NODE_PROJECT=\"tsconfig-for-webpack.json\" webpack serve --config webpack/webpack.dev.ts",
     "fetch-deps": "node --loader ts-node/esm ./util/fetch_deps.ts",
     "tsc-no-emit": "node_modules/.bin/tsc --noEmit",
     "lint": "node_modules/.bin/eslint --max-warnings=0 ./",
```

Comment:
I think you need to `s/node_modules\/.bin\///` here.  Things like `npm run lint` don't work with node v16 if it says `node_modules/.bin/eslint` instead of just `eslint` on its own.

Reply:
Current config works fine both on my computer and ci.

Reply:
Sorry, I don't know what's wrong, but the current config doesn't work for me.

Does it work if you remove `node_modules/.bin/` as well? If so, then maybe that's a solution that will more consistently work for everybody.  I know in the past somebody was suggesting we should remove that path anyway.

Reply:
```
enne@thuse MINGW64 ~/cactbot (Trim21-npm-lock-v2)
$ npm run test

> cactbot@0.23.10 test
> node_modules/.bin/mocha

'node_modules' is not recognized as an internal or external command,
operable program or batch file.
```

Reply:
Haha, thank you.  I'm sorry, I don't know why it works for you but not for me!